.github/workflows/pull-requests.yml

```diff
@@ -4,15 +4,14 @@ on:
     branches:
       - '**'
 
-jobs:
-  PR-Greeting:
-    name: Greeting Message to user
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/first-interaction@v1
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          pr-message: "Congratulations on making your first PR! :confetti_ball: If you haven't already, check out our [Contributing Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md) and [PR Reporting Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/PR-guidelines.md) to ensure that you are following our guidelines for contributing and creating PR."
+#jobs:
```

Comment:
Yasharth. We'll still need the line

`jobs:`

Reply:
ok sir changing